src/service/link-rewriter/link-rewriter-manager.js

```diff
@@ -0,0 +1,231 @@
+import {AmpEvents} from '../../amp-events';
```

Comment:
The only thing that has changed here is the name of the file, the name of the class, the comments and the type annotation. Code hasn't changed